lib/msal-core/tests/MSAL.spec.ts

```diff
@@ -29,6 +29,7 @@ describe('Msal', function (): any {
 
     var RESOURCE_DELIMETER = '|';
```

Comment:
Travis build is failing. Take latest from dev once the types/jasmine issue is fixed there.